### ISSUE TITLE
Review and update README

### ISSUE THREAD

**Issue by jonespm** [tl-its-umich-edu/unizin-validation#14]
Kudos for updating the README. It probably needs a thorough read through once we've tested. I'll create a separate issue.

_Originally posted by @ssciolla in https://github.com/tl-its-umich-edu/unizin-validation/pull/11_

**Comment by ssciolla** [comment]
Thanks for creating the issue, I am going to wait on this until we resolve the email issue (#10), since you're editing this file and some details we'll want to cover in the README might change.
